core/postfix/conf/main.cf

```diff
@@ -112,4 +112,7 @@ milter_default_action = tempfail
 ###############
 # Extra Settings
 ###############
-
+sender_canonical_maps = tcp:localhost:10001
```

Comment:
I would put these under a previous section specific to SRS instead of just extra settings.